static/_redirects

```diff
@@ -0,0 +1,10 @@
+# Redirect rules for Netlify
+# Although splats are supported, e.g. /* /new/path/:splat, regex isn't yet
+
+# Serve schema files located in the cnabio/cnab-spec repository
+/v1/bundle.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/bundle.schema.json
+/v1/claim.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/claim.schema.json
+/v1/definitions.schema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/definitions.schema.json
+/v1/dependenciesschema.json https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/dependenciesschema.json
```

Comment:
This 404's for me because it's missing a period before schema.json

https://raw.githubusercontent.com/cnabio/cnab-spec/master/schema/dependencies.schema.json

Reply:
Whoops!  Thank you for testing all of them 🙇 

I'll look at adding some super simple CI to cnab-spec to iterate thru all the schemas and test their availability from cnab.io after this is merged.